app/src/main/java/org/lineageos/selfie/MainActivity.kt

```diff
@@ -727,20 +760,51 @@ class MainActivity : AppCompatActivity() {
     }
 
     private fun openGallery() {
-        dismissKeyguardAndRun {
-            sharedPreferences.getLastSavedUri()?.let { uri ->
-                listOf(MediaStore.ACTION_REVIEW, Intent.ACTION_VIEW).forEach {
+        sharedPreferences.getLastSavedUri().let { uri ->
+            // If the Uri is null attempt to lunch non secure-gallery
```

Comment:
launch

Reply:
this is what years of building AOSP does to you, mixing lunch and launch.